tests/Issues/CountArrayLongToShort/Fixture/count_to_empty_array_compare.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
this commited `.php` file fixture can be removed, it seems due to you cut process under running `vendor/bin/phpunit` then use git add